test/evergreen.yml

```diff
@@ -2061,10 +2061,6 @@ tasks:
       - func: "format test"
         vars:
           extra_args: checkpoints=1 leak_memory=0 mmap=1 file_type=row compression=zlib huffman_value=1
-      # FIXME-WT-6668: temporarily disable lower isolation level test
-      # - func: "format test"
-      #   vars:
-      #     extra_args: checkpoints=1 leak_memory=0 mmap=1 file_type=row isolation=random transaction_timestamps=0
```

Comment:
Should we retain/re-enable this test to cover non-timestamp (standalone wiredtiger) run with lower isolation levels (read)?

Reply:
@lukech, this test no longer makes sense -- when `timestamps=0` is set, lower-level isolation is included in all cases, not just when specifically configured (and the `isolation` configuration has been removed).

Reply:
Alright, thanks for the explanation - it makes sense to me now. 
Just hope to make sure we still have coverage for lower isolation levels.